src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -350,6 +350,18 @@ trait EntityComponent {
       }
     }
 
+    object DeleteEntityQuery extends RawSqlQuery {
+      val driver: JdbcDriver = EntityComponent.this.driver
+
+      def deleteEntityAction(workspaceId: UUID) =
```

Comment:
Move this inside `EntityAndAttributesRawSqlQuery` and rename to `deleteActionForWorkspace`